backend/requirements/dev.txt

```diff
@@ -71,13 +92,15 @@ pip-tools==7.3.0
 platformdirs==2.4.0
     # via pylint
 psycopg2==2.9.1
-    # via -r requirements.in/base.txt
+    # via -r requirements.in\base.txt
```

Comment:
I wonder whether a backslash will cause problems on unix systems... But if this runs in the containers,  it's probably fine.